### ISSUE TITLE
Missing updates for sbt-sonatype and sbt-pgp

### ISSUE THREAD

**Issue by albuch** [fthomas/scala-steward#1014]
After #1011 was fixed I'm still missing two pull requests for plugins defined in:
https://github.com/albuch/sbt-dependency-check/blob/master/project/sonatype.sbt
sbt-pgp is available in version 2.0.0
sbt-sonatype is available in version 3.8

**Comment by fthomas** [comment]
Are these updates reported by sbt-updates' `dependencyUpdates`?

**Comment by albuch** [comment]
Not all of them are reported and dependencyUpdates seems to ignore the project settings (like excludes and failing the build) when checking plugins:
```
$ sbt dependencyUpdates; reload plugins; dependencyUpdates; reload return
....
[info] Found 1 dependency update for sbt-dependency-check
[info]   commons-collections:commons-collections : 3.2.2 -> 20040102.233541
...
[info] Found 3 dependency updates for project
[info]   com.jsuereth:sbt-pgp                    : 1.1.2-1                     -> 2.0.1-M3       
[info]   commons-collections:commons-collections : 3.2.2                       -> 20040102.233541
[info]   org.scala-lang:scala-library:provided   : 2.10.7            -> 2.13.1                   
```

**Comment by fthomas** [comment]
sbt-pgp is not updated because of #956. You need to bump the version to 2.0.0 manually for now.

Regarding sbt-sonatype, could it be the case that 3.8 is not available for sbt 0.13?

**Comment by albuch** [comment]
You are absolutely right, I even have an Issue to upgrade to sbt 1.x for sbt-sonatype upgrade but totally forgot that 2.3 was the last compatible version with 0.13 ... Thanks for the clarification.
